### PR TITLE
Sharing

### PR DESCRIPTION
**Pull request by eliascodes**
Changes necessary to make this config easily shareable.

Tested this locally using `npm link` and adding an `.eslintrc.json` file to a new project with the following contents:

```json
{
  "extends": "dwyl"
}
```

Updated README with instructions.

All thats then left to be done is to publish to `npm`. Current version is `0.0.1` which will be the initial version unless someone has a problem with it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
   },
   "devDependencies": {
     "eslint": "^3.5.0",
     "tape": "^4.6.0"
-  }
+  },
+  "peerDependencies": {
+    "eslint": ">=3.5.0"
+  },
+  "contributors": [
```
Comment (jrans) [inline package.json]: gitter handle may be better! hehe

Comment (jrans) [inline package.json]: @nelsonic included too!

File: package.json [hunk 2]
```diff
@@ -1,12 +1,23 @@
 {
-  "name": "goodparts",
+  "name": "eslint-config-dwyl",
```
Comment (jrans) [inline package.json]: do we want to double check name with everyone? brings us back to #192 , the name of this module will be a good indication

Comment (jrans) [inline package.json]: `dwyl-goodparts` `eslint-goodparts` `eslint-dwyl` all possible contenders too

File: README.md [hunk 3]
```diff
@@ -56,8 +56,17 @@ http://eslint.org/docs/about/
...
+```
+$ npm i --save-dev eslint-config-dwyl
+```
+And then in your `.eslintrc` just add the following:
+```
+{
+  "extends": "dwyl"
+}
+```
+Alternatively, you can also add this to the `"eslintConfig"` field in your `package.json`.
```
Comment (jrans) [inline README.md]: I'm lazy can you provide the exact linting command

Comment (eliascodes) [inline README.md]: Made this explicit in 1e7ab4c

File: package.json (hunk 1 above)
Comment (eliascodes) [inline package.json]: changed to github handles and added @nelsonic

**Comment by jrans** [comment]
Before merging name of package needs to be decided but then ready, @shouston3 @SimonLab @nelsonic thoughts?

File: package.json (hunk 2 above)
Comment (shouston3) [inline package.json]: They all sound fine

**Comment by eliascodes** [comment]
After speaking to @nelsonic we decided we wanted a zero configuration setup, so I've added a simple bin file that will run the eslint CLI with our config.

File: README.md [hunk 4]
```diff
@@ -56,8 +56,16 @@ http://eslint.org/docs/about/
...
-
-> This is where _you_ come in @SimonLab ... :wink:
+Simply install our configuration
+```
+$ npm i --save-dev goodparts
+```
+Then you can add the following script to your `package.json`:
+```
+{
+  "lint": "node_modules/.bin/goodparts path/to/files/for/linting"
```
Comment (nelsonic) [inline README.md]: @eliascodes _could_ we specify a `bin` script in `package.json` ?
see: https://docs.npmjs.com/files/package.json#bin

Comment (eliascodes) [inline README.md]: I have https://github.com/dwyl/goodparts/pull/229/commits/82796e608ca7e4d81fb216c523787651d48a964d

Comment (nelsonic) [inline README.md]: Looks like a good start. thanks!

**Comment by nelsonic** [comment]
thanks @eliascodes !! 👍 
❤️ ✅ 🚢

**Comment by iteles** [comment]
Did we just go with `goodparts` for now or did you guys actually make a call on the name?

**Comment by eliascodes** [comment]
@iteles we went with it for now because we own [this](https://www.npmjs.com/package/goodparts) already. If anyone feels strongly about the name we can open another discussion.

**Pull request merged**